frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -1060,9 +1060,6 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
             <Row justify="center" align="middle">
               <Button
                 onClick={this.showAddVolumeLayerModal}
-                style={{
```

Comment:
The button was cutting off the text. I guess another side-effect of the new font. I don't think a strict width is necessary here.